crypto_build.sh

```diff
@@ -3,7 +3,7 @@
 # crypto package
 PKG_NAME="github.com/onflow/flow-go/crypto"
 
-VERSION="v0.24.4"
+VERSION="v0.24.7"
```

Comment:
update to the latest version, otherwise it would cause inconsistent flow-go version